dispatcher.go

```diff
@@ -276,6 +294,7 @@ func (d *Dispatcher) Start() error {
 		allStarted []transport.Lifecycle
 	)
 
+	d.log.Info("Starting up.")
```

Comment:
Happy to go with the group consensus on this.

I prefer to make logs complete sentences, since they have well-understood criteria for distinguishing coherent thoughts from word salad. After lots of time poring over other people's logs, I find that valuable.